packages/user-agent/src/user-agent.ts

```diff
@@ -213,7 +223,7 @@ export class Web5UserAgent implements Web5ManagedAgent {
     throw new Error('Not implemented');
   }
 
-  async sendDidRequest(_request: SendDidRequest): Promise<DidResponse> {
+  async sendDidRequest(_request: DidRequest): Promise<DidResponse> {
     throw new Error('Not implemented');
```

Comment:
how does this relate to 
rpc-client.sendDidRequest() ?

Reply:
Its a good question and I think the use case for sending requests is different for DIDs than it is with DWNs.

With `web5.did.resolve()` there really isn't a scenario where you would send the DID resolution request to another remote agent.

Even once `web5.did.create()` is introduced, I can't imagine a scenario in which a `Web5UserAgent` would send a request to another agent asking it to create the DID (and keys).

So the only scenario would be something like a `Web5ProxyAgent` communicating with something like a desktop Identity Agent... In that case, in `packages/proxy-agent/src/proxy-agent.ts` you would expect that all of the calls to `processDidRequest` are actually calls to `Web5RpcClient.sendDidRequest()`, which then forwards the request over the network.